sdcm/rest/storage_service_client.py

```diff
@@ -0,0 +1,56 @@
+from typing import Optional
+
+from fabric.runners import Result
+from requests import PreparedRequest
+
+from sdcm.cluster import BaseNode
+
+from sdcm.rest.rest_client import RestClient
+
+
+class StorageServiceClient:
+    def __init__(self, node: BaseNode):
+        self._node = node
+        self._host = "localhost:10000"
+        self._remoter = self._node.remoter
+        self._endpoint = "storage_service"
+        self.__client = RestClient(host=self._host, endpoint=self._endpoint)
```

Comment:
Please convert it into:

```
@cached_property
def _client(self):
    RestClient(host=self._host, endpoint=self._endpoint)
```

Reply:
I changed the implementation here to use inheritance rather than composition, so we don't need this property anymore.